samples/net/lwm2m_client/sample.yaml

```diff
@@ -7,6 +7,16 @@ tests:
     depends_on: netif
     platform_allow: qemu_x86 frdm_k64f pinnacle_100_dvk
     tags: net lwm2m
+    extra_configs:
```

Comment:
Should we have a CI compilation of the sample without these options set too, meaning one compilation with these and one without?

Reply:
Probably makes sense to have the sample built in the default configuration, I'll add a custom variant for all objects build.